src/funfuzz/js/shared/testing-functions.js

```diff
@@ -18,8 +18,20 @@ function fuzzTestingFunctionsCtor(browser, fGlobal, fObject)
 
   function enableGCZeal()
   {
-    var level = rnd(17);
-    if (browser && level == 9) level = 0; // bug 815241
+    // As of m-c 405243:02aa9c921aed 2018-02-26
+    // https://hg.mozilla.org/mozilla-central/file/02aa9c921aed/js/src/gc/GC.cpp#l1000
+    var level = rnd(19);
+
+    // Generate the second level.
+    // ref https://hg.mozilla.org/mozilla-central/file/02aa9c921aed/js/src/gc/GC.cpp#l1001
+    var level_two = rnd(19);
+    if (level_two == 3) level_two = 0; // level_two 3 does not exist, so repurpose it
+    if (level_two == 5) level_two = 0; // level_two 5 does not exist, so repurpose it
+    if (level_two == 6) level_two = 0; // level_two 6 does not exist, so repurpose it
```

Comment:
This is fine, but if you don't want to add weight to `level_two == 0` you could also do something like:

    var level_two = rnd(19 - 3); // 3 levels disabled below
    if (level_two >= 3) level_two++;
    if (level_two >= 5) level_two++;
    if (level_two >= 6) level_two++;

Reply:
This will mean that if `level_two` comes out as:

```
0 has no change
1 has no change
2 has no change
3 -> 4
4 -> 5 -> 6 -> 7
5 -> 6 -> 7 -> 8
6 -> 7 -> 8 -> 9
7 -> 8 -> 9 -> 10
etc.
15 -> 16 -> 17 -> 18
```

I understand this now, good idea.